src/main/java/de/doubleslash/usb_led_matrix/view/ConfigurationView.java

```diff
@@ -430,9 +432,26 @@ void refreshButton() {
       }
    }
```

Comment:
why is the class field `timeLightsTurnedOffNow` still only LocalTime? why is it not LocalDateTime? Do we need this like it?

Reply:
Done is now LocalDateTime